cypress/integration/plugins/security-dashboards-plugin/change_tenant_successfully.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CURRENT_TENANT } from '../../../utils/commands';
+import tenantDescription from '../../../fixtures/plugins/security-dashboards-plugin/tenants/testTenant.json';
+import { switchTenantTo } from './switch_tenant';
+
+const tenantName = 'tenant';
+
+if (Cypress.env('SECURITY_ENABLED')) {
+  describe('Switch tenants when visiting copied links: ', () => {
+    before(() => {
+      cy.server();
+
+      cy.createTenant(tenantName, tenantDescription);
+    });
+    it('Checks that the tenant switcher can switch tenants despite a different tenant being present in the tenant query parameter.', function () {
+      CURRENT_TENANT.newTenant = tenantName;
```

Comment:
Q: Why changing `defaultTenant` globally? Can we pass `currentTenant` via visit command's `options` to overwrite if `currentTanant` needs to be changed in individual test? Something like this:

```typescript
newOptions.qs = { 
  ...newOptions.qs,
  security_tenant: newOptions.currentTenant ?? CURRENT_TENANT.defaultTenant,
};
```